src/kvstore/Part.cpp

```diff
@@ -433,9 +433,19 @@ nebula::cpp2::ErrorCode Part::cleanup() {
   LOG(INFO) << idStr_ << "Clean rocksdb part data";
   auto batch = engine_->startBatchWrite();
   // Remove the vertex, edge, index, systemCommitKey, operation data under the part
+
+  const auto& kvPre = NebulaKeyUtils::kvPrefix(partId_);
+  auto ret =
+      batch->removeRange(NebulaKeyUtils::firstKey(kvPre, 128), NebulaKeyUtils::lastKey(kvPre, 128));
```

Comment:
why `128`? Is it a magic number?

Reply:
relax....it's not certain yet...

Reply:
I am not sure how to handle it …… For example, if we want to delete part 3's kv data. The prefix is `06 03 00 00` in hex, can we delete the range by `[06 03 00 00, 06 03 00 01]`?

Reply:
why not just part_id plus 1 as the last key?

Reply:
sorry, forget that part_id is encoded in little endian.